packages/web/src/pages/artist-coins-launchpad-page/components/ArtistCoinsTable.tsx

```diff
@@ -125,7 +125,7 @@ const renderVolume24hCell = (cellInfo: CoinCell) => {
   return (
     <Text variant='body' size='m'>
       {walletMessages.dollarSign}
-      {formatCount(coin.v24hUSD)}
+      {coin.v24hUSD.toFixed(2)}
```

Comment:
does this break larger numbers? what if the volume is 10000 for example? we won't get 10K anymore as we would with `formatCount`

Reply:
yep ur totally right